src/autocomplete/databases/yql/grammar/README.md

```diff
@@ -0,0 +1,72 @@
+When new grammar is arrived, it is a must to do next steps before generating parsers:
```

Comment:
Before generating parsers for new grammar version following steps should be done: